lib/libraries/c3/extensions/default-date-format.js

```diff
@@ -0,0 +1,32 @@
+module.exports = function(startDate, endDate){
```

Comment:
Do c3 extensions require this sort of setup with returning mixed types like this? I think we should ideally avoid returning mixed types like this (`Function` sometimes, other times `String`).